src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -476,10 +477,26 @@ private static unsafe void Avx2Decode(ref byte* srcBytes, ref byte* destBytes, b
             destBytes = dest;
         }
 
+        private static Vector128<byte> SimdShuffle(Vector128<byte> left, Vector128<byte> right, Vector128<byte> maskf)
```

Comment:
I'd add a note that this helper can be deleted if/when https://github.com/dotnet/runtime/issues/63331 is implemented